marketing/copy.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Marketing Copy
+description: Text to use on sites which promote Mathesar
+published: true
+date: 2023-02-23T00:00:00.000Z
+tags: 
+editor: markdown
+dateCreated: 2023-02-23T00:00:00.000Z
+---
+
+## Tagline
+
+### Technical
+
+> Mathesar is an open source web application providing an intuitive spreadsheet-like interface to PostgreSQL databases.
+
+
+### Non-technical
+
+> Mathesar is an application for managing structured data from an intuitive spreadsheet-like web interface.
```

Comment:
This seems like it would be somewhat scary for non-technical users. I don't think they will know what "structured data" means. We should try to speak in the "voice of the customer". I recommend taking inspiration from this page: https://mathesar.org/non-technical-expert.html, which was based on Myra's research into non-technical users.

Reply:
Also see above note about personality.